src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/CommonSteps.java

```diff
@@ -7,7 +7,7 @@
 import org.umssdiplo.automationv01.core.managepage.SHLogin;
 import org.umssdiplo.automationv01.core.utils.LoadPage;
 
-public class LoginSteps {
+public class CommonSteps {
```

Comment:
by the moment you will need to use just one step definitions

Reply:
Ingeniero, como usted nos comentó solo deberíamos mantener un  archivo de step definitions, por eso decidimos implementar uno con un nombre genérico. Lo que hicimos fue reemplazar el nombre del que ya teníamos, nada mas, solo tenemos un step definitions.